util/warnp.c

```diff
@@ -51,12 +52,18 @@ warn(const char * fmt, ...)
 	va_list ap;
 
 	va_start(ap, fmt);
-	fprintf(stderr, "%s", (name != NULL) ? name : "(unknown)");
-	if (fmt != NULL) {
-		fprintf(stderr, ": ");
-		vfprintf(stderr, fmt, ap);
+	if (syslog_use == 0) {
+		/* Print to console. */
```

Comment:
s/console/stderr/.  The *console* is something quite different.

Reply:
done